engines/stark/visual/text.cpp

```diff
@@ -109,7 +115,7 @@ void VisualText::createTexture() {
 
 	// Render the lines to the surface
 	for (uint i = 0; i < lines.size(); i++) {
-		font->drawString(&surface, lines[i], 0, scaledLineHeight * i, scaledRect.width(), _color);
+		font->drawString(&surface, lines[i], 0, scaledLineHeight * i - 1, scaledRect.width(), _color);
```

Comment:
Are you sure about this change? This means y = -1 for the first line which does not seem correct.